client/src/components/common/ReviewStars.js

```diff
@@ -41,7 +41,7 @@ const FillStar = ({ size = 24, star = 0, fillPerc = 0, ratingId, checked, handle
         }}
       ></span>
       <label htmlFor={`${ratingId}_${star}`} className="pd-visual-hidden">Star {star}</label>
-      <input id={`${ratingId}_${star}`} ref={starRadio} type="radio" checked={checked} onChange={handleChange} value={star}/>
+      <input id={`${ratingId}_${star}`} ref={starRadio} type="radio" checked={checked} onChange={handleChange}  readOnly={handleChange ? false : true} value={star}/>
```

Comment:
If an onChange handler(handleChange) is not provided to this component, for example if the rendered stars are static and not user selectable, then we need to set the readOnly attribute to true. Otherwise react throws a warning.